content/exporters/custom-exporter/Node.js/Metrics.md

```diff
@@ -73,25 +73,34 @@ export class MyConsoleStatsExporter implements StatsEventListener {
 And now to test it out as we would in a typically linked program, let's create a `expample.js` file:
 
 {{<highlight javascript>}}
-var opencensus = require('@opencensus/core');
-var stats = new opencensus.Stats();
+const { globalStats, AggregationType, TagMap } = require('@opencensus/core');
 
 // Let's create an instance of our just created exporter
-var exporter = new MyConsoleStatsExporter();
+const exporter = new MyConsoleStatsExporter();
 // And register it
-stats.registerExporter(exporter);
+globalStats.registerExporter(exporter);
 
 // Let's create a measure
-var measure = stats.createMeasureInt64('my/measure', "1");
+const measure = globalStats.createMeasureInt64('my/measure', "1");
 // our tags
-var tags = {myTagKey: 'myTagValue'};
-// a view
-var view = stats.createView('my/view', measure, 2, ['myTagKey'], 'my view');
+const myTagKey = { name: "myTagKey" };
+const tags = new TagMap();
+tags.set(myTagKey, { value: "myTagValue" });
+
+// Create and Register the view
+const view = globalStats.createView(
+  'my/view', 
+  measure, 
+  AggregationType.LAST_VALUE, 
+  [myTagKey], 
+  'my view'
+);
+globalStats..registerView(view);
```

Comment:
This doesn't look right `globalStats..`, double dots "..". Were you able to run this code?

Reply:
Yes, I ran the example and it works as expected.